frontend/stub/__files/response-details-XP.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "Thiel, Wintheiser and Vandervort",
+  "id": "{{request.path.[2]}}",  
+  "addresses": [
+    {
+      "streetAddress": "123 Fake DR",
+      "country": {
+        "value": "CA",
+        "text": "Canada"
+      },
+      "province": {
+        "value": "BC",
+        "text": "British Columbia"
+      },
+      "city": "Fakest City",
+      "postalCode": "A0A0A0",
+      "index": 0,
+      "locationName": "Mailing address"
+    }
+  ],
+  "contacts": [
+    {
+      "contactType": {
+        "value": "P",
+        "text": "Person"
```

Comment:
Do we need to change this due to the changes on the "Contact Type Codes".
I guess that might be the case, and there are other samples of 
"contactType": {
    "value": "P",
    "text": "Person"
}
throughout the codebase that we might need to update.

Reply:
Yeah, but assuming that the data is coming from an external source (in this case a faked bc registries) and not all data can be trusted (for example the location to contact association), even if we keep the wrong data, our code should take care of it by not allowing the user to proceed.